sass/components/_asset-card.scss

```diff
@@ -2,12 +2,13 @@ $asset-card-padding: 0.4rem;
 
 .asset-card {
     display: grid;
+    max-width: 94vw;
+    aspect-ratio: auto 10/9;

```

Comment:
Is `auto 10/9` a valid value here?
https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio
It seems like it should be 10/9?
In chrome at least, it seems to be behaving just as `auto` which doesn't seem like what we want?